lib/pbench/server/api/resources/query_apis/elasticsearch_api.py

```diff
@@ -25,12 +25,10 @@ def post(self):
 
         try:
             # query Elasticsearch
+            url = f"{self.elasticsearch}/{json_data['indices']}"
             if "params" in json_data:
-                url = (
-                    f"{self.elasticsearch}/{json_data['indices']}?{json_data['params']}"
-                )
-            else:
-                url = f"{self.elasticsearch}/{json_data['indices']}"
+                if "ignore_unavailable" in json_data["params"]:
+                    url = f"{url}?ignore_unavailable={json_data['params']['ignore_unavailable']}"
 
             if "payload" in json_data:
                 es_response = requests.post(url, json=json_data["payload"])
```

Comment:
A better (cleaner, and more general) mechanism might be to pass through a distinct `params` to the server API to use in the `requests` POST directly. That way we don't need to worry about `?` vs `&`, and so forth... which is the point of separating the query parameters in the requests interfaces in the first place. Rather than trying to hardcode them in the URI string...

Reply:
yes I thought about that but this is an open es endpoint disguised as `<server>/elasticsearch` to hide the actual endpoint, we anyway not gonna have this endpoint in the long run. Right now dashboard does some es queries directly to es and to support that for a while this is just a hot fix for correctly formatting the es URI

Reply:
I'm actually not sure we don't want to continue support for this endpoint. This would enable more generic open-ended ES queries for clients than we support via canned "endpoint" queries for advanced exploration, without opening direct ES server access. We'll have to figure out how to balance that against user-based security, but I'd thought about something like a "direct ES" Role that we could allow for specific users.

Anyway; again, as a point fix, this is fine, and we can worry about how to generalize/support something like this going forward in a separate context.